src/main/java/frc/robot/RobotContainer.java

```diff
@@ -79,13 +87,68 @@ public Command getDriveCommand() {
     return driveCommand;
   }
 
+   public Command getturnleft () {
+     return left;
+   }
+
+  public Command getturnright () {
+    return Right;
+  }
   public Joystick getDriverJoystick() {
     return driverController;
   }
 
   public Joystick getCopilotJoystick() {
     return copilotController;
   }
+  public Command getgoforward () {
```

Comment:
naming convention:
should be ` public Command getGoForward () {
    return goForward;
  }`